tools/Vrops.py

```diff
@@ -359,7 +374,11 @@ def get_project_ids(target: str, token: str, uuids: list, collector: str) -> (li
                                      data=json.dumps(payload),
                                      params=querystring,
                                      verify=False,
-                                     headers=headers)
+                                     headers=headers,
+                                     timeout=60)
```

Comment:
Question still persists, do we really need to wait `60s`?

Reply:
What's @richardtief opionen about this?
It seems 60s was chosen for the longest running requests in `_get_chunk`, which I believe is a reasonable conservative value, i.e. between 'premature cutoff' and 'respond before scrape timeout'.
With adding it to all other requests we just make sure it doesn't get stuck, without potentially alter the current behaviour.

Reply:
To get stats about the other requests we should export their `.elapsed.total_seconds()` and deduce the needed timeouts. That's a topic for a separate PR, though.

Reply:
When looking through the overall scrape times I can find `65s` in the bigger regions as a maximum for the entire scrape, not just one request.
As the requests here would sum up we are way over the `scrape_interval`, given every request takes the same time. I'd suggest to go with `30`.

Reply:
Agreed